caluma/caluma_data_source/data_sources.py

```diff
@@ -86,3 +87,8 @@ def try_get_data_with_fallback(self, user, question, context):
                 raise e
             return self.default
         return new_data
+
+    def on_copy(
+        self, old_answer: Answer, new_answer: Answer, old_value: Tuple[str, str]
+    ) -> Tuple[Optional[str], Optional[str]]:
+        return old_value
```

Comment:
Can I have a docstring here to describe what's happening / what's supposed to happen in subclasses?

Or just update the class's docstring instead: What can this be useful for? What considerations to take into account? How to implement a certain behaviour?